tests/end2end/features/yankpaste.feature

```diff
@@ -20,6 +20,19 @@ Feature: Yanking and pasting.
         Then the message "Yanked URL to primary selection: http://localhost:(port)/data/title.html" should be shown
         And the primary selection should contain "http://localhost:(port)/data/title.html"
 
+    Scenario: Yanking URLs with ref and UTM data to clipboard
+        When I open http://kikolani.com/blog-post-promotion-ultimate-guide?utm_source=kikolani&utm_medium=320banner&utm_campaign=bpp
```

Comment:
You can't open URLs like that here (because it'd make the test flaky, and slower) - instead, you simply load paths like `data/hello.txt` and a local webserver will serve files from [the `tests/end2end/data` directory](https://github.com/The-Compiler/qutebrowser/tree/master/tests/end2end/data).

You can probably just use `data/hello.txt?utm_source=...` as the parameters should simply be ignored by the server.